src/store/getters.ts

```diff
@@ -298,4 +308,34 @@ export default {
     }
     return false;
   },
+  nftAssetsByCollection(...context: GetterContext): NFTAsset[] {
+    const { getters } = rootGetterContext(context);
+    const { nftAssetsByAccount } = getters;
+    let nftAssetsByCollection: NFTAsset[] = [];
+    nftAssetsByCollection = Object.values(nftAssetsByAccount).reduce((accum, nftAssets) => {
```

Comment:
The `reduce` function is not used correctly here. Please check https://codeburst.io/how-to-reduce-array-to-a-single-type-11cfe2bcae66